node/utils.go

```diff
@@ -310,13 +311,22 @@ func createVolumeDir(hostPath, containerSpec, imageSpec, userSpec, permissionSpe
 	createVolumeDirMutex.Lock()
```

Comment:
initial startup is now 1 minute 30 for initial start of zenoss.resmgr on AWS.  Mariadb averages 30 seconds  :(